shardingsphere-ui-backend/src/main/java/org/apache/shardingsphere/ui/util/CenterRepositoryFactory.java

```diff
@@ -91,7 +102,7 @@ private static CenterConfiguration convert(final CenterConfig config) {
         CenterConfiguration result = new CenterConfiguration(config.getInstanceType(), new Properties());
         result.setServerLists(config.getServerLists());
         result.setNamespace(config.getNamespace());
-        result.getProps().put("digest", config.getDigest());
+        result.getProperties().put("digest", config.getDigest());
```

Comment:
The lastest code of master branch has been changed from `Properties` to `Props`, please check.

Reply:
May be need `install` the local master branch first.